src/titiler/core/titiler/core/factory.py

```diff
@@ -320,27 +320,6 @@ def register_routes(self):
         if self.add_viewer:
             self.map_viewer()
 
-    ############################################################################
-    # /bounds
-    ############################################################################
-    def bounds(self):
-        """Register /bounds endpoint."""
-
-        @self.router.get(
-            "/bounds",
-            response_model=Bounds,
-            responses={200: {"description": "Return dataset's bounds."}},
-        )
-        def bounds(
-            src_path=Depends(self.path_dependency),
-            reader_params=Depends(self.reader_dependency),
-            env=Depends(self.environment_dependency),
-        ):
-            """Return the bounds of the COG."""
-            with rasterio.Env(**env):
-                with self.reader(src_path, **reader_params.as_dict()) as src_dst:
-                    return {"bounds": src_dst.geographic_bounds}
```

Comment:
The `bounds` endpoints isn't really useful alone. Users can use the `/tilejson.json` or `/info.geojson` to get more geo information about the dataset 

Note: `geographic_bounds` has been remove in rio-tiler and replaced with `get_geographic_bounds`

Reply:
changed my mind 🙈